### PR TITLE
CCE resource exports kube_config_raw

### LINKED ISSUE #469: Feature request: CCE resource exports raw kube config content
Hiiii,
For the CCE resource, besides exporting individual kube config variables like certificate_clusters/server etc (the current behaviour), it'd be nice if it can also export the whole kube config in some encoded format.

**Use case**:
In a continuous deployment pipeline, we setup the cluster access by creating the local ~/.kube/config file within our CD container. While it is possible to do so by executing the `kubectl config`  command multiple times with needed kubeconfig credentials, one can simply `cat` the raw config content if he has access to it.

For consideration,  Azure Kubernetes Service resource exports `kube_config_raw`: https://www.terraform.io/docs/providers/azurerm/r/kubernetes_cluster.html#kube_config_raw

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
fixes #469

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_cce_cluster_v3.go [hunk 1]
```diff
@@ -147,6 +147,10 @@ func resourceCCEClusterV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"kube_config_raw": {
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cce_cluster_v3.go:150]: Add `StateFunc` function will be better.

File: huaweicloud/resource_huaweicloud_cce_cluster_v3.go [hunk 2]
```diff
@@ -323,7 +327,12 @@ func resourceCCEClusterV3Read(d *schema.ResourceData, meta interface{}) error {
 	d.Set("security_group_id", n.Spec.HostNetwork.SecurityGroup)
 	d.Set("region", GetRegion(d, config))
 
-	cert, err := clusters.GetCert(cceClient, d.Id()).Extract()
+	r := clusters.GetCert(cceClient, d.Id())
+	kubeConfigRaw, _ := jsonMarshal(r.Body)
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cce_cluster_v3.go]: the error result should be judged here if the r.Body is abnormal.

Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_cluster_v3.go]: done

File: huaweicloud/resource_huaweicloud_cce_cluster_v3.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_cluster_v3.go:150]: It seems the StateFunc won't be called until d.Set( ) succeed，but r.Body can't be set directly as TypeString

**Pull request merged**